eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxo.hs

```diff
@@ -272,8 +271,10 @@ feesOK ::
     Core.Tx era ~ Alonzo.ValidatedTx era,
     -- "collateral" to get inputs to pay the fees
     HasField "collateral" (Core.TxBody era) (Set (TxIn (Crypto era))),
+    HasField "txrdmrs" (Core.Witnesses era) (Redeemers era),
```

Comment:
This seems to be missing from the original PR..but probably not relevant

Reply:
yea, this was made obsolete by https://github.com/input-output-hk/cardano-ledger/pull/2901